bin/phase-helpers.sh

```diff
@@ -557,7 +557,7 @@ unpack() {
 						"with EAPI '${EAPI}'. Instead use 'txz'."
 				fi
 				if ___eapi_unpack_supports_txz; then
-					XZ_OPT="-T$(___makeopts_jobs)" tar xof "${srcdir}${x}" || die "${myfail}"
+					XZ_OPT="-T$(___makeopts_jobs) --memlimit-mt-decompress=50%" tar xof "${srcdir}${x}" || die "${myfail}"
```

Comment:
6a47cc7bcf49b7c17fdca31969b6d6aeff84332c calls out the decompress option -- but what I don't get yet is there's already a default limit for multithreaded decompression (which is part of why I didn't add it to begin with).

This is mentioned both in the man page and you can see it with:
```console
$ xz --info-memory
Hardware information:
  Amount of physical memory (RAM):  64,215 MiB (67,334,234,112 B)
  Number of processor threads:      32

Memory usage limits:
  Compression:                      Disabled
  Decompression:                    Disabled
  Multi-threaded decompression:     16,054 MiB (16,833,558,528 B)
  Default for -T0:                  16,054 MiB (16,833,558,528 B)
````

That's less than 50%. So, while we could add an option, adding it with a value of 50% doesn't seem to do much (if anything it increases the limit).

What am I missing?